### DISCUSSION TITLE
School switched email domains, but GitHub still requires the old one?

### DISCUSSION THREAD

**Discussion by LyubomirT** [community/community#102832] [GitHub Education]
### Hi everyone,

I verified myself using my school email when first received access to the Education Pack. However, a few months ago, the school portal got hacked, and the hacker received administrative permissions, hence using this email is unsafe at the moment. That's why my school has decided to switch emails and increase the protection level. As I noticed, GitHub still requires the old email domain to verify for the Education Pack (I don't need to reverify yet, I'm just concerned for my schoolmates who may want to register as well).

Is there a way I can apply the new domain of my school to GitHub Education? Any help is truly appreciated.

**Comment by phuccoder2000** [comment]


**Reply by LyubomirT** [reply]
Okay! Thank you for the information, GitHub support might indeed be helpful in this case.
